### ISSUE TITLE
Google Assistant integration

### ISSUE THREAD

**Issue by gonzojive** [gonzojive/heatpump#5]
See 

- https://developers.google.com/assistant/smarthome/overview
- https://developers.google.com/assistant/smarthome/guides/thermostat
- https://developers.google.com/assistant/smarthome/concepts/devices-traits

It looks like the fan coil units can use the FanSpeed, OnOff, TemperatureSetting, TemperatureControl, and maybe more.

**Comment by njmckenzie** [comment]
Have you had success connecting to the air handlers?

**Comment by gonzojive** [comment]
Yes, the code that's checked in connects to the air handlers, which I call "fan coil units" in this repository.

There isn't any Google Assistance integration yet, though. The fan coil units don't come with such integration - it would need to be added by this project.

**Comment by njmckenzie** [comment]
Sorry for being dense but I'm not seeing any code which connects to the fan coil units (probably operator error).  The connection to the outdoor unit is clear in the code as well as the register values that correspond to that outdoor unit.  I don't see any corollaries for the indoor equipment.

**Comment by gonzojive** [comment]
I just merged the air handler code into the main branch. Before now it was on the "fancoil" branch.

This will bring up a gRPC server for retrieving the status and controlling the units:

https://github.com/gonzojive/heatpump/blob/main/fancoil/cmd/fancoil_status/fancoil_status.go

See this, too

https://github.com/gonzojive/heatpump/blob/main/proto/fancoil/fancoil.proto

I have a program that sends commands to the server a few times a day to adjust the fan speed and temperature.

**Comment by njmckenzie** [comment]
Dang your good!
